ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -242,6 +242,9 @@ - (void)handleControlEventFromSender:(UIControl *)sender withEvent:(UIEvent *)ev
 #if DEBUG
 static void checkMethodSignatureAgainstTypeEncodings(SEL selector, NSMethodSignature *signature, const std::vector<const char *> &typeEncodings)
 {
+  if (selector == NULL)
```

Comment:
Argh, wait wait wait. Sorry, I missed that there's a missing {} here. We strictly include {} everywhere in our codebase to avoid things like goto fail; goto fail; :P